### PR TITLE
feat(ci): replace circleci with github actions

### PR DESCRIPTION
**Pull request by adrianfalleiro**
## What

Replaces CircleCI with Github Actions

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  ci:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Build container image
+        run: docker build --build-arg NODE_ENV=development -t "$CONTAINER_IMAGE" .
```
Comment (Thexumaker) [inline .github/workflows/ci.yaml:19]: this doesn't need a test env?

**Review by jeremy-ke — APPROVED** [review]
just need to potentially remove status check in settings -> branch protections + stop building in circleci. thanks!

**Review by Thexumaker — APPROVED** [review]
LGTM
just updated branch rules and turned off circleCI webhook, might need to push an update to see the effect

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (adrianfalleiro) [reply .github/workflows/ci.yaml:19]: Nope just needs a debug build of the native module 🎉

**Pull request merged**